custom/icds_reports/utils.py

```diff
@@ -457,3 +458,14 @@ def create_pdf_file(pdf_hash, pdf_context):
 
 def icds_pre_release_features(user):
     return toggles.ICDS_DASHBOARD_REPORT_FEATURES.enabled(user.username)
+
+
+def indian_formatted_number(amount):
+    amount = Decimal(str(amount))
+    amount, thousands = divmod(amount, 1000)
+    groups = [thousands]
+    while amount:
+        amount, group = divmod(amount, 100)
+        groups.insert(0, group)
+
+    return ','.join(map(str, groups))
```

Comment:
no big deal but this is the type of function that would be nice to have a couple unit tests for to document it and make sure it's working right. also it looks like it will fail hard on invalid inputs - confirming that's desired?